google-built-opentelemetry-collector/docs/examples/deployment/cos/cloud-init-config-example.yaml

```diff
@@ -148,4 +124,9 @@ write_files:
           - googlecloud
       telemetry:
         metrics:
-          address: localhost:8888
+          readers:
```

Comment:
We should add a descriptive comment to this section similar to the descriptive comments on all the sections above.